planning march 2021/userStoriesSummary.md

```diff
@@ -0,0 +1,145 @@
+## Features
+
+### Org employee
+
+- [ ] Create user without an organisation - 1d
+- [ ] Join organisation - 1
+- [ ] Edit user details - 2
+- [ ] Should only see/edit application for their company and user - 1
+- [ ] Do updates on applications after "changes required" - 1
+- [ ] Get notified by email when "changes required" / "submitted" / "outcome changes" - 1
+- [ ] See application with different outcomes - 1
+- [ ] Request admin permission in company - 3
+- [ ] Apply for drug registration - 1
```

Comment:
I wonder what story we should link front end plugins (maybe this one ?).
- Table
- Document Upload
- Free Text, etc..

Reply:
Yeah I think an epic for remaining plugins is a good idea.
Also -- you think we need a Date Picker element? (I'd say so). Semantic UI doesn't have a built-in module, but this package looks like it would make it fairly straightforward:
https://www.npmjs.com/package/react-semantic-ui-datepickers